app/messaging/application/submit-claim.js

```diff
@@ -26,12 +26,23 @@ const submitClaim = async (message) => {
       if (application.dataValues.claimed) {
         return sendMessage({ state: alreadyClaimed }, submitClaimResponseMsgType, applicationResponseQueue, { sessionId: message.sessionId })
       }
-      const res = await updateByReference({ reference, claimed: true, statusId: 4, updatedBy: 'admin', data })
+
+      const applicaitonsCount = await getApplicationsCount()
+
+      let statusId = 9
+      if (applicaitonsCount % 5 === 0) {
+        statusId = 5
+      }
+
+      const res = await updateByReference({ reference, claimed: true, statusId, updatedBy: 'admin', data })
 
       const updateSuccess = isUpdateSuccessful(res)
 
       if (updateSuccess) {
         await sendFarmerClaimConfirmationEmail(application.dataValues.data.organisation.email, reference)
+      }
+
+      if (updateSuccess && statusId === 9) {
```

Comment:
Can probably do it later but might be good to have some kind of config file or enum with the statusId's so its easier for someone looking for the first time to know what the status id refers to.